homeassistant/components/vlc/media_player.py

```diff
@@ -79,11 +78,11 @@ def update(self):
         """Get the latest details from the device."""
         status = self._vlc.get_state()
         if status == vlc.State.Playing:
-            self._state = MediaPlayerState.PLAYING
+            self._attr_state = MediaPlayerState.PLAYING
         elif status == vlc.State.Paused:
-            self._state = MediaPlayerState.PAUSED
+            self._attr_state = MediaPlayerState.PAUSED
         else:
-            self._state = MediaPlayerState.IDLE
+            self._attr_state = MediaPlayerState.IDLE
         self._media_duration = self._vlc.get_length() / 1000
         position = self._vlc.get_position() * self._media_duration
```

Comment:
Other properties in this PR could have the same change. Let's keep it consistent on how integration works.